zegami_sdk/test.py

```diff
@@ -72,3 +72,11 @@ def test_ensure_token_user_login(self):
                 token=None, allow_save_token=True
             )
             self.assertNotEqual(os.path.getsize(self.local_token_path), 0)
+
+
+@unittest.mock.patch.dict(os.environ, {'ALLOW_INSECURE_SSL': 'True'})
+class TestSdkUtilVerifySSLFalse(TestSdkUtil):
```

Comment:
This should ideally run the TestSDKUtil with the environment patched - however it doesn't seem to cover the extra lines